src/cdk/scrolling/virtual-scroll-viewport.scss

```diff
@@ -0,0 +1,20 @@
+cdk-virtual-scroll-viewport {
```

Comment:
Add comments here that explain how these different elements fit together since it's pretty fundamental to the way that the virtual-scroller works?